build/startup/index.md

```diff
@@ -160,11 +160,11 @@ Before the 0,0 point and Z endstop locations are set, the physical locations of
 
 ### V1, Legacy:
 
-The Z endstop should be located at close to max X position.  Home X and Y with `G28 X Y`  and then traverse just Y to locate a Z endstop position at the maximum X travel that will still trigger the endstop.  Lock down the Z endstop at that position.
+The Z endstop should be located close to max X position.  Home X and Y with `G28 X Y`  and then traverse just Y to locate a Z endstop position at the maximum X travel that will still trigger the endstop.  Lock down the Z endstop at that position.
```

Comment:
this should say **as** close to max as possible

Reply:
Addressed